test/test.js

```diff
@@ -3301,6 +3303,24 @@ describe('Page', function() {
       }
     });
   });
+
+  describe('Response', function() {
+    it('specifies when response comes from cache and service worker', async({page, server}) => {
+      // Use known site that uses server caching and service worker caching.
+      let responses = [];
+      page.on('response', response => {
+        if (response.url.includes('.js'))
+          responses.push(response);
+      });
+      await page.goto('https://www.chromestatus.com/features', {waitUntil: 'networkidle2'});
+      expect(responses[0].fromDiskCache).toBe(false);
+      expect(responses[0].fromServiceWorker).toBe(false);
+      responses = [];
+      response = await page.reload({waitUntil: 'networkidle2'});
```

Comment:
looks like `response` is being assigned to the scope but not used

Reply:
done